synth/validator/reward.py

```diff
@@ -81,17 +81,11 @@ def reward(
     ]
     real_price_path = [entry["price"] for entry in intersecting_real_price]
 
-    try:
-        score, detailed_crps_data = calculate_crps_for_miner(
-            np.array(predictions_path),
-            np.array(real_price_path),
-            simulation_input.time_increment,
-        )
-    except Exception as e:
-        bt.logging.error(
-            f"Error calculating CRPS for miner {miner_uid} with prediction_id {miner_prediction_id}: {e}"
-        )
-        return -1, [], [], miner_prediction_id
+    score, detailed_crps_data = calculate_crps_for_miner(
```

Comment:
even if we fix the issue, why can't we keep the exception handler in case something else happens in the crps calculation we don't know yet?